subprojects/gradle/instrumentation-tests/src/main/kotlin/com/avito/instrumentation/InstrumentationTestsTask.kt

```diff
@@ -123,6 +125,10 @@ abstract class InstrumentationTestsTask @Inject constructor(
             reportConfig
         )
 
+        val proguardMappings = project.tasks.withType<ProguardConfigurableTask>()
+            .filter { it.mappingFile.isPresent }
+            .map { it.mappingFile.get().asFile }
```

Comment:
I consider we need to avoid using `get` on configuration time.
You should use `flatMap { it.mappingFile }` and then at execution time you would extract provider value by `get()`

Reply:
It is not configuration time. It is inside `doWork` of the task. But now I'm refactoring and moving this code into a plugin.

Reply:
This code was rewritten